exonum-java-binding/core/src/main/proto/testkit.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.testkit;
+
+option java_package = "com.exonum.binding.testkit";
```

Comment:
As it is in core, it must be in "com.exonum.binding.core.testkit" — to avoid proliferation of split packages.

May also sprinkle "internal" as the last part — to communicate it's not a part of the API 🙃